reactioncommerce/reaction-next-starterkit
# Integrate the FinalReviewCheckoutAction

Resolves #269 
Impact: major
Type: feature

### Summary
Lay the groundwork for integrating the `FinalReviewCheckoutAction` with the starterkit.

TODO: 
- [x] Integrate UI with backend

## Testing
1. Place on order
2. Verify order shows up in reaction
3. Verify cart token and cartId were removed from client
4. Verify corresponding cart was removed from DB
